js/scenes/NumberOfGuestScreen/NumberOfGuestScreenContainer.js

```diff
@@ -16,27 +20,44 @@ class NumberOfGuestScreenContainer extends React.Component {
     };
   }
   static navigationOptions = {
-    title: 'Guest number',
-    header: ({ navigate }) => ({
-      // Render a button on the right side of the header
-      // When pressed switches the screen to edit mode.
+    header: ({ navigate, dispatch, goBack }) => ({
+      style: { height: 0 ,margin: 0, padding: 0, },
+      title: <CreatePotluckProgressBar title='Number of Guests' progressNumber={1} />,
       right: (
-        <Button
-          title={'Done'}
+        <NavigationArrow
           onPress={() => navigate('FoodPlanningScreen')}
         />
       ),
-      backTitle: null,
+      left: (
+        <NavigationArrow
+          backArrow
+          onPress={() => goBack(null)}
+        />
+      ),
     }),
   };
+
   onTextChange = (text) => {
-    this.setState({ count: text });
-  }
-  increase = () => {
-    this.setState({ count: this.state.count + 1 });
+    if (text === '') {
+      this.setState({ count: 0 });
+      return;
+    }
+    const reg = /(^\d+$)/;
+    if (reg.test(text)) {
```

Comment:
We're checking for numeric here, right? What if we just provided the numeric keyboard?